services/core-api/app/api/mines/reports/resources/mine_reports.py

```diff
@@ -193,61 +192,6 @@ def post(self, mine_guid):
             if mine_report_contacts:
                 mine_report.mine_report_contacts = mine_report_contacts
 
-        # TODO: remove following with CODE_REQUIRED_REPORTS feature flag (submissions, if submissions)
```

Comment:
Goodbye CODE_REQUIRED_REPORTS feature flag!
I think SonarCloud is yelling at me about some unused variables that I just didn't quite believe I could delete. I _did_ try to delete some unused imports and then it turned out they were not actually unused so I restored them.